src/dcd_mapping/schemas.py

```diff
@@ -164,12 +167,15 @@ class ScoreAnnotation(BaseModel):
     This model defines what an individual mapping instance looks like in the final JSON.
     """
 
-    pre_mapped: vrs_v1_schemas.VariationDescriptor | vrs_v1_schemas.Haplotype | Allele | Haplotype
-    post_mapped: vrs_v1_schemas.VariationDescriptor | vrs_v1_schemas.Haplotype | Allele | Haplotype
-    vrs_version: VrsVersion
     mavedb_id: StrictStr
-    relation: Literal["SO:is_homologous_to"] = "SO:is_homologous_to"
-    score: float | None
+    relation: Literal[
+        "SO:is_homologous_to"
+    ] = "SO:is_homologous_to"  # TODO this should probably be None if pre_mapped is false?
```

Comment:
That makes sense to me. Does pre-mapped being false imply that people should just ignore this? Is it a lot of effort to change?